core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -2165,11 +2165,12 @@ private DefaultServerConfig buildServerConfig(List<ServerPort> serverPorts) {
 
         final ServerErrorHandler errorHandler;
         if (this.errorHandler == null) {
-            errorHandler = ServerErrorHandler.ofDefault();
+            errorHandler = new CorsServerErrorHandler(ServerErrorHandler.ofDefault());
```

Comment:
nit: What do you think about using `?:`, so we don't have to call `CorsServerErrorHandler` constructor twice?

```
final ServerErrorHandler errorHandler =
    new CorsServerErrorHandler(
        this.errorHandler == null ? ServerErrorHandler.ofDefault()
                                  : this.errorHandler.orElse(ServerErrorHandler.ofDefault())); 
```

Reply:
That sounds great. I've incorporated the changes you recommended into the code.